package/network/utils/wwan/files/wwan.sh

```diff
@@ -93,11 +93,11 @@ proto_wwan_setup() {
 	uci_set_state network $interface dat_device "$dat_device"
 
 	case $driver in
-	qmi_wwan)	proto_qmi_setup $@ ;;
-	cdc_mbim)	proto_mbim_setup $@ ;;
-	sierra_net)	proto_directip_setup $@ ;;
-	comgt)		proto_3g_setup $@ ;;
-	*cdc_ncm)	proto_ncm_setup $@ ;;
+	qmi_wwan)		proto_qmi_setup $@ ;;
```

Comment:
with the current change, if I keep the original tab, the last line (cdc_ether|*cdc_ncm) will be not alligned.

Are you sure that this is the best practice?